src/components/AppShelf/AppShelf.tsx

```diff
@@ -123,100 +119,36 @@ const AppShelf = (props: any) => {
-              value={idx.toString()}
-            >
-              <Accordion.Control>{category}</Accordion.Control>
-              <Accordion.Panel>{getItems(category)}</Accordion.Panel>
-            </Accordion.Item>
-          ))}
-          {/* Return the item for all services without category */}
-          {noCategory && noCategory.length > 0 ? (
-            <Accordion.Item
-              style={{
-                background: `rgba(${colorScheme === 'dark' ? '32, 33, 35,' : '255, 255, 255,'} \
-            ${(config.settings.appOpacity || 100) / 100}`,
-                borderColor: `rgba(${colorScheme === 'dark' ? '32, 33, 35,' : '233, 236, 239,'} \
-            ${(config.settings.appOpacity || 100) / 100}`,
-              }}
-              key="Other"
-              value="Other"
-            >
-              <Accordion.Control>{t('accordions.others.text')}</Accordion.Control>
-              <Accordion.Panel>{getItems()}</Accordion.Panel>
-            </Accordion.Item>
-          ) : null}
-          {downloadEnabled ? (
-            <Accordion.Item
-              style={{
-                color: `rgba(${colorScheme === 'dark' ? '32, 33, 35,' : '255, 255, 255,'} \
-                ${(config.settings.appOpacity || 100) / 100}`,
-                background: `rgba(${colorScheme === 'dark' ? '32, 33, 35,' : '255, 255, 255,'} \
-            ${(config.settings.appOpacity || 100) / 100}`,
-                borderColor: `rgba(${colorScheme === 'dark' ? '32, 33, 35,' : '233, 236, 239,'} \
-            ${(config.settings.appOpacity || 100) / 100}`,
-              }}
-              key="Downloads"
-              value="Your downloads"
-            >
-              <Accordion.Control>{t('accordions.downloads.text')}</Accordion.Control>
-              <Accordion.Panel>
-                <Paper radius="lg" style={{ position: 'relative' }}>
-                  {torrentEnabled && (
-                    <>
-                      <Text>{t('accordions.downloads.torrents')}</Text>
-                      <ModuleMenu module={TorrentsModule} hovered />
-                      <TorrentsComponent />
-                    </>
-                  )}
-                  {usenetEnabled && (
-                    <>
-                      <Text mt="md">{t('accordions.downloads.usenet')}</Text>
-                      <ModuleMenu module={UsenetModule} hovered />
-                      <UsenetComponent />
-                    </>
-                  )}
-                </Paper>
-              </Accordion.Panel>
-            </Accordion.Item>
-          ) : null}
-        </Accordion>
-      </Stack>
-    );
-  }
   return (
     <Stack>
+      <Accordion
+        variant="separated"
+        radius="lg"
+        order={2}
+        multiple
+        value={toggledCategories}
+        onChange={(state) => {
+          setToggledCategories([...state]);
+        }}
+      >
+        {categoryList.map((category, idx) => (
+          <Accordion.Item
+            style={{
```

Comment:
I think we can't extract this to ``createStyles``, but consider extracting it to a function with the appropriate TS type